Assets/Scripts/Game/PlayerActivate.cs

```diff
@@ -387,6 +392,37 @@ void Update()
             }
         }
 
+        private void ActivateBulletinBoard(RaycastHit hit, DaggerfallBulletinBoard bulletinBoard)
+        {
+            if(currentMode == PlayerActivateModes.Info)
+            {
+                // Check if close enough to Activate
+                if(hit.distance > MobileNPCActivationDistance)
+                {
+                    DaggerfallUI.SetMidScreenText(HardStrings.youAreTooFarAway);
+                    return;
+                }
+
+                // Get news
+                var bulletinBoardMessage = GameManager.Instance.TalkManager.GetNewsOrRumorsForBulletinBoard();
+
+                // format message
+                var tokens =  new TextFile.Token[]
+                {
+                            new TextFile.Token(TextFile.Formatting.JustifyCenter, null),
+                            new TextFile.Token(TextFile.Formatting.Text, GameManager.Instance.PlayerGPS.CurrentLocation.Name),
+                            new TextFile.Token(TextFile.Formatting.JustifyCenter, null),
+                            new TextFile.Token(TextFile.Formatting.NewLineOffset, null),
+                            new TextFile.Token(TextFile.Formatting.Text, string.Empty),
+                            new TextFile.Token(TextFile.Formatting.NewLineOffset, null),
+                            new TextFile.Token(TextFile.Formatting.Text, bulletinBoardMessage),
```

Comment:
weren't sure where to add tokenization of the message, so I added it here.

Reply:
Hmm, I would assume the messages from classic would be in the text.rsc and would have the formatting as well as macros for things like the town name etc
Did you look?

Reply:
hmm, didn't take a look there. I'll do it now, see if I can use it.

Reply:
@ajrb ok so I took a look at https://en.uesp.net/wiki/Daggerfall:TEXT.RSC_indices , and generated the CSV file from TEXT.RSC.  There do exist 2 City bulletin messages sections, but these only contain the rumor itself, not the full "{cityname} \n {rumor}" text, for example: 
"1475"	"%nrn has declared that all witches[0xFD][0xFC] are to be burnt at the stake.[0xFD]

the only thing which kinda could be is is this:

0028 = "%hnt"

but I'm not sure what it means. 

Apart from these I couldn't find anything in TEXT.RSC

Reply:
%abc are macros and are handled by a MacroHelper class. For instance %nrn is the name of the local noble.

Maybe these messages are used with a different one. I would have to look.

Reply:
The way Chimp implemented this matches classic. In classic, the region name is always displayed on top of all local rumors, if any. Then, an empty line is displayed between the region name and the first rumor, and so on.

Reply:
That's fine, other multi-part messages are handled by entries in the rsc text but this could be different - just thought I'd raise it.